packages/SwingSet/src/kernel/state/vatKeeper.js

```diff
@@ -173,6 +174,35 @@ export function makeVatKeeper(
     return harden(options);
   }
 
+  /**
+   * @param {SwingSetCapData} newVPCD
+   */
+  function setVatParameters(newVPCD) {
+    insistCapData(newVPCD);
+    const key = `${vatID}.vatParameters`;
+    // increment-before-decrement to minimize spurious rc=0 checks
+    for (const kref of newVPCD.slots) {
+      incrementRefCount(kref, `${vatID}.vp`);
```

Comment:
Nit, keep the tag fully descriptive?

```suggestion
      incrementRefCount(kref, `${vatID}.vatParameters`);
```

Reply:
Yeah, ok, I did a quick scan of our other tags, and we're kinda 50/50 with short (`enq` for enqueue, `vk|clist` for vat-keeper -managed clists) and long (`create-vat-event`, `external`, etc). But given that there's only one of these per vat, and we don't actually record them anyways (unless maybe some debug flag is on), there's no need to be terse. Will do.